.gitignore

```diff
@@ -104,4 +104,5 @@ dist/
 /data
 
 # Patch files
-*.patch
\ No newline at end of file
+*.patch
+scratch/
```

Comment:
Didn't @rajadain say he global `.gitignored` this? (sorry for the drive-by, it seemed like a cool feature so I wanted to check it out)

Reply:
Here is how I've set it up:

https://github.com/rajadain/dotfiles/blob/master/.gitconfig#L22
https://github.com/rajadain/dotfiles/blob/master/.gitignore#L7

Reply:
yea, and I've got enough random files from testing this pr that sticking them in there is easy lol

Reply:
<details>

<summary>
Oh you can get pretty crazy once you start using it.
</summary>

```
> tree ~/dev/mmw/scratch/
/Users/ttuhinanshu/dev/mmw/scratch/
├── Failing_TR55.json
├── Failing_TR55_FeatureCollection.geojson.txt
├── Failing_TR55_Union.geojson.txt
├── LittleNeshaminy_MapshedJob.json
├── LittleNeshaminy_MapshedJob_AnalyzeResult.json
├── LittleNeshaminy_MapshedJob_result.json
├── LittleNeshaminy_MapshedJob_result2.json
├── LittleNeshaminy_SummaryJob.json
├── LittleNeshaminy_SummaryJob_AnalyzeResult.json
├── LittleNeshaminy_SummaryJob_result.json
├── Little_Nishemany\ with\ DRB__Current_Conditions.gms.txt
├── Little_Nishemany\ without\ DRB__Current_Conditions.gms.txt
├── MapshedJob_DRB_Large.json
├── MapshedJob_DRB_Medium.json
├── MapshedJob_DRB_Small.json
├── MapshedJob_DRB_VeryLarge.json
├── MapshedJob_GroundwaterNitrogen.json
├── MapshedJob_NHD.json
├── MapshedJob_RasterAverage.json
├── MapshedJob_RasterGroupedAverage.json
├── MapshedJob_RasterGroupedCount.json
├── MapshedJob_RasterGroupedSum.json
├── NevadaSimple.json
├── PettyIslandDelawareRiver_MapshedJob_AnalyzeResult.json
├── PettyIslandDelawareRiver_SummaryJob_AnalyzeResult.json
├── SchuykillRWD67000.json
├── Schuylkill.json
├── SchuylkillHUC08_MapshedJob.json
├── SchuylkillHUC08_SummaryJob.json
├── SouthCarolina.json
├── SummaryJob.json
├── bigcz
│   ├── CUAHSI-soapui-project.xml
│   ├── cinergi-test.json
│   ├── cinergi-test.txt
│   ├── cuahsi-world.json
│   ├── cuahsi_baseline.json
│   ├── cuahsi_test.json
│   ├── hydroshare-philadelphia-20170728.json.txt
│   ├── hydroshare-philadelphia.json.txt
│   ├── hydroshare-portland.json.txt
│   └── hydroshare-seattle-20170728.json.txt
├── collections-api
│   ├── awc-request.json
│   ├── awc-response.json
│   ├── crum-creek-huc12.geojson
│   ├── fetch-less
│   │   ├── huc10-conestoga-envelope-intersect.geojson
│   │   └── huc10-conestoga-multipolygon-intersect.geojson
│   ├── huc10-conestoga.geojson
│   ├── huc8
│   │   ├── nlcd-request-huc8.json.txt
│   │   ├── nlcd-response-huc8-new.json.txt
│   │   ├── nlcd-response-huc8.json.txt
│   │   ├── nlcd-soils-request-huc8.json.txt
│   │   ├── soil-request-huc8.json.txt
│   │   ├── soil-response-huc8-new.json.txt
│   │   ├── soil-response-huc8.json.txt
│   │   ├── soil-texture-request-huc8.json.txt
│   │   ├── soil-texture-response-huc8-new.json.txt
│   │   └── soil-texture-response-huc8.json.txt
│   ├── huc8-schuylkill.geojson
│   ├── nlcd-kfactor-mismatch-request.json.txt
│   ├── nlcd-kfactor-request.json
│   ├── nlcd-kfactor-response.json
│   ├── nlcd-request-huc10.json
│   ├── nlcd-request-huc8.json
│   ├── nlcd-request.json
│   ├── nlcd-response-huc8-new.json
│   ├── nlcd-response-huc8.json
│   ├── nlcd-response.json
│   ├── nlcd-soil-census-request.json
│   ├── nlcd-soil-census-response.json
│   ├── nlcd-soils-request-huc10.json
│   ├── nlcd-soils-request-huc12.json
│   ├── nlcd-soils-request-huc8.json
│   ├── nlcd-soils-request.json
│   ├── nlcd-soils-response.json
│   ├── nlcd-streams-crum-creek-huc12-request.json
│   ├── nlcd-streams-crum-creek-huc12-response.collections-api.json
│   ├── nlcd-streams-crum-creek-huc12-response.json
│   ├── nlcd-streams-crum-creek-huc12.zip
│   ├── nlcd-streams-request.json
│   ├── nlcd-streams-response.collections-api.json
│   ├── nlcd-streams-response.json
│   ├── node_modules
│   │   └── lodash -> ../../../../../../../usr/local/lib/node_modules/lodash
│   ├── pr64
│   │   ├── new
│   │   │   ├── nlcd-response-huc10.json
│   │   │   ├── nlcd-response-huc8.json
│   │   │   ├── nlcd-response.json
│   │   │   ├── nlcd-soil-census-response.json
│   │   │   ├── nlcd-soils-response-huc10.json
│   │   │   ├── nlcd-soils-response-huc12.json
│   │   │   ├── nlcd-soils-response-huc8.json
│   │   │   ├── nlcd-soils-response.json
│   │   │   ├── soil-response-huc8.json
│   │   │   ├── soil-response.json
│   │   │   └── soil-texture-response-huc8.json
│   │   └── old
│   │       ├── nlcd-response-huc10.json
│   │       ├── nlcd-response-huc8.json
│   │       ├── nlcd-response.json
│   │       ├── nlcd-soil-census-response.json
│   │       ├── nlcd-soils-response-huc10.json
│   │       ├── nlcd-soils-response-huc12.json
│   │       ├── nlcd-soils-response-huc8.json
│   │       ├── nlcd-soils-response.json
│   │       ├── soil-response-huc8.json
│   │       ├── soil-response.json
│   │       └── soil-texture-response-huc8.json
│   ├── raster-grouped-average
│   │   ├── awc-request.json
│   │   └── nlcd-kfactor-request.json
│   ├── raster-grouped-average.tgz
│   ├── raster-grouped-count
│   │   ├── nlcd-request-huc10.json
│   │   ├── nlcd-request-huc8.json
│   │   ├── nlcd-request-huc8.json.txt
│   │   ├── nlcd-request.json
│   │   ├── nlcd-soil-census-request.json
│   │   ├── nlcd-soils-request-huc10.json
│   │   ├── nlcd-soils-request-huc12.json
│   │   ├── nlcd-soils-request-huc8.json
│   │   ├── nlcd-soils-request-huc8.json.txt
│   │   ├── nlcd-soils-request.json
│   │   ├── soil-request-huc8.json
│   │   ├── soil-request-huc8.json.txt
│   │   ├── soil-request.json
│   │   ├── soil-texture-request-huc8.json
│   │   └── soil-texture-request-huc8.json.txt
│   ├── raster-grouped-count.tgz
│   ├── soil-request-huc8.json
│   ├── soil-request.json
│   ├── soil-response-huc8-new.json
│   ├── soil-response-huc8.json
│   ├── soil-response.json
│   ├── soil-texture-request-huc8.json
│   ├── soil-texture-response-huc8-new.json
│   ├── soil-texture-response-huc8.json
│   └── test.json
├── cuahsi-search-results.patch
├── dev.hydroshare.org.http
├── echo.sh
├── geojson-upload
│   ├── feature.geojson
│   ├── featurecollection.geojson
│   ├── multipolygon.geojson
│   ├── parse-fail.geojson
│   └── polygon.geojson
├── geoprocessing-helpers.txt
├── huc-query-performance.sql
├── hydroshare
│   ├── xBrgajUHMkRY.cpg
│   ├── xBrgajUHMkRY.dbf
│   ├── xBrgajUHMkRY.prj
│   ├── xBrgajUHMkRY.shp
│   └── xBrgajUHMkRY.shx
├── long-sidebar-text.patch.txt
├── ls
│   ├── crop-cult.dbf
│   ├── cty_animals.dbf
│   ├── cty_animals.prj
│   ├── cty_animals.sbn
│   ├── cty_animals.sbx
│   ├── cty_animals.shp
│   ├── cty_animals.shx
│   └── pasture.dbf
├── mapshed-latency
│   ├── analyze-2537-after.json
│   ├── analyze-2537-patterops.json
│   ├── analyze-2537.json
│   ├── analyze.json
│   ├── explain-2537.sql
│   ├── explain.sql
│   ├── huc08-schuylkill.geojson
│   ├── huc08-schuylkill.json
│   ├── rwd-schuylkill.geojson
│   ├── rwd-schuylkill.json
│   └── upload
│       ├── huc08-schuylkill.json.txt
│       └── rwd-schuylkill.json.txt
├── mapshed_data.json
├── micro.html
├── ms_county_animals.20160610174906.sql.gz
├── ms_county_animals.20160816114300.sql.gz
├── ms_weather_station_old.sql
├── overlapping-test
│   ├── awc-request.json
│   ├── awc-response.json
│   ├── nlcd-kfactor-request.json
│   ├── nlcd-kfactor-response.json
│   ├── nlcd-request-huc10.json
│   ├── nlcd-request-huc8.json
│   ├── nlcd-request.json
│   ├── nlcd-response-huc10.json
│   ├── nlcd-response-huc8.json
│   ├── nlcd-response.json
│   ├── nlcd-soil-census-request.json
│   ├── nlcd-soil-census-response.json
│   ├── nlcd-soils-request-huc10.json
│   ├── nlcd-soils-request-huc12.json
│   ├── nlcd-soils-request-huc8.json
│   ├── nlcd-soils-request.json
│   ├── nlcd-soils-response-huc10.json
│   ├── nlcd-soils-response-huc12.json
│   ├── nlcd-soils-response-huc8.json
│   ├── nlcd-soils-response.json
│   ├── nlcd-streams-request.json
│   ├── nlcd-streams-response.json
│   ├── overlap-test-request.json
│   ├── overlap-test-response.json
│   ├── soil-request-huc8.json
│   ├── soil-request.json
│   ├── soil-response-huc8.json
│   ├── soil-response.json
│   ├── soil-texture-request-huc8.json
│   └── soil-texture-response-huc8.json
├── overlapping-test.zip
├── overlapping-test2
│   ├── awc-request.json
│   ├── awc-response.json
│   ├── nlcd-kfactor-request.json
│   ├── nlcd-kfactor-response.json
│   ├── nlcd-request-huc10.json
│   ├── nlcd-request-huc8.json
│   ├── nlcd-request.json
│   ├── nlcd-response-huc10.json
│   ├── nlcd-response-huc8.json
│   ├── nlcd-response.json
│   ├── nlcd-soil-census-request.json
│   ├── nlcd-soil-census-response.json
│   ├── nlcd-soils-request-huc10.json
│   ├── nlcd-soils-request-huc12.json
│   ├── nlcd-soils-request-huc8.json
│   ├── nlcd-soils-request.json
│   ├── nlcd-soils-response-huc10.json
│   ├── nlcd-soils-response-huc12.json
│   ├── nlcd-soils-response-huc8.json
│   ├── nlcd-soils-response.json
│   ├── nlcd-streams-request.json
│   ├── nlcd-streams-response.json
│   ├── soil-request-huc8.json
│   ├── soil-request.json
│   ├── soil-response-huc8.json
│   ├── soil-response.json
│   ├── soil-texture-request-huc8.json
│   └── soil-texture-response-huc8.json
├── ppt-request.json
├── production-failing-shape.geojson
├── remove-soil-texture.patch.txt
├── schuylkill_huc08
│   ├── MapshedJob_RasterAverage.json
│   ├── MapshedJob_RasterGroupedAverage.json
│   ├── MapshedJob_RasterGroupedCount.json
│   ├── MapshedJob_RasterGroupedCount_HUC08.json.txt
│   ├── MapshedJob_RasterGroupedSum.json
│   └── schuylkill_huc08.geojson
├── schuylkill_huc10
│   ├── MapshedJob_RasterAverage.json
│   ├── MapshedJob_RasterGroupedAverage.json
│   ├── MapshedJob_RasterGroupedCount.json
│   ├── MapshedJob_RasterGroupedSum.json
│   ├── mapshed.data
│   └── schuylkill_huc10.geojson
├── ssugro.json
├── test.gms.txt
├── test.json.txt
├── test.sh
├── time-mapshed.patch.txt
└── time.txt

20 directories, 252 files
```
</details>

Reply:
how did you make that code block collapsed?

Reply:
GitHub Magic :octocat: ✨ 

```markdown
<details>

<summary> THE TEXT YOU WANT TO SHOW / BE CLICKABLE </summary>

THE TEXT YOU WANT TO HIDE

</details>
```